libmve/mveplay.cpp

```diff
@@ -0,0 +1,481 @@
+/*
+ * Copyright (C) 2002-2024 D2X Project
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#define AUDIO
```

Comment:
Is `AUDIO` ever not defined? It would be better if we could disable sound at runtime rather than compile-time

Reply:
Probably this will go for second iteration. There already some issues related to separated sound playback for video and game - latter one has volume control, and in result video plays on maximum volume ignoring config and cmdline-options. I'll create issue about merging these two sound devices into one.

Reply:
sounds good